client/src/AppBundle/Resources/assets/javascripts/main.js

```diff
@@ -95,6 +95,16 @@ $(document).ready(function () {
       new ButtonToggler().init($el)
     })
   }
+
+  const $submitButtons = document.querySelectorAll('[data-module="opg-toggleable-submit"]')
+
+  if ($submitButtons !== null) {
+    $submitButtons.forEach(function ($el) {
+      $el.addEventListener('click', function ($e) {
+        this.classList.add('opg-submit-link--disabled')
```

Comment:
Maybe add `this.classList.add('govuk-button--disabled')` as well as the `opg-submit-link--disabled` styling so when the button is pressed once it's also greyed out 👍🏻